src/wp-admin/css/media.css

```diff
@@ -1429,6 +1424,14 @@ audio, video {
 		height: auto;
 		width: 100%;
 	}
+
+	.media-item .edit-attachment.copy-to-clipboard-container {
+		flex-direction: column;
+	}
+	.copy-to-clipboard-container .success {
+		position: relative;
+		top: 10px;
+	}
```

Comment:
Is this the intended placement of the 'success' message with a viewport under 480px?

![Screenshot 2024-05-22 at 12 10 14](https://github.com/WordPress/wordpress-develop/assets/1682452/8911b92b-4614-406c-b13b-51e45cf7959b)

Reply:
Yes. For 480, we can move to right side as usual. But below 420, its kind of cutting out of view port. So below 420 we should put it like this.. But as there was no separate media queries below 480, i added in that one.

Reply:
@afercia Do you want me to add separate media query for 420 or this will work?

Reply:
@afercia I have added separate media query below 375 for this. We cannot add line-height: normal as its moving edit text on left due to increase in height of right area. So added more value on top which looks good now.

Reply:
> @afercia Do you want me to add separate media query for 420 or this will work?

I think it's OK to use existing media queries 👍🏽

Reply:
> We cannot add line-height: normal as its moving edit text on left due to increase in height of right area.

What I tried on my side should work:

When the media query kicks, add the following CSS for the `.attachment-tools` element:
```
display: flex;
align-items: baseline;
```

and add this for the `.success` element:

```
line-height: normal;
margin-top: 10px;
```

This way, the 'edit' text should stay aligned.

Screenshot:

![Screenshot 2024-05-22 at 12 48 09](https://github.com/WordPress/wordpress-develop/assets/1682452/a6a5d5a8-c044-460e-af65-5b3246580053)